assets/src/domain/eventEditor/ui/tickets/ticketsList/tableView/ticketsListTableRow.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { __ } from '@wordpress/i18n';
+
+import { displayDatesFilter } from '@appLayout/espressoTable/utils';
+import { DisplayDates } from '@edtrInterfaces/shared/types';
+import { ENTITY_LIST_DATE_TIME_FORMAT } from '@appConstants/dateFnsFormats';
+import { getBackgroundColorClassName, status } from '@sharedEntities/tickets/helpers';
+import { InlineEditText } from '@appInputs/InlineEditInput';
+import { shortenGuid } from '@appServices/utilities/text';
+import { Ticket } from '@edtrServices/apollo/types';
+import TicketActionsMenu from '@edtrUI/tickets/ticketsList/actionsMenu/TicketActionsMenu';
+import { useMoneyDisplay } from '@appServices/utilities/money';
+import { useTicketMutator } from '@edtrServices/apollo/mutations';
+
+import '@application/ui/styles/root/entity-status.css';
```

Comment:
oooo i didn't know this worked for css too :D

Reply:
I also did not knew, but before this is had a very looong `'../.../////...../../././.././.././.\...\.\.\..\`